src/main/resources/org/sonatype/nexus/ci/nxrm/v3/CreateTagBuilder/help-tagAttributesJson.html

```diff
@@ -0,0 +1,18 @@
+<!--
+
+    Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+
+    This program is licensed to you under the Apache License Version 2.0,
+    and you may not use this file except in compliance with the Apache License Version 2.0.
+    You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the Apache License Version 2.0 is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+-->
+<div>
+  Optional metadata for the tag in JSON format. These attributes will be merged with those contained in the
```

Comment:
I seem to remember the team having a discussion about using either a file or JSON on the invocation but not both. Is this a change from that? Also, which takes priority if the same information is provided in both?

Reply:
In implementation it seemed simple enough to offer both options; took liberties of deciding that any directly entered JSON text would be the advanced option, and take priority, my thought being a user may have a simple file that they like to use for all tags, and perhaps modify the build to augment those defaults per build.

Reply:
But yes, this was a deviation from the initial discussion. Open to change if this seems off base.